docs/src/layouts/MainLayout.vue

```diff
@@ -213,19 +203,23 @@ export default defineComponent({
 $footer-columns-md-min: 6;
 $footer-columns-sm-min: 4;
 $adjust-header-viewport: 860px;
-$hide-social-links-viewport: 687px;
+$hide-social-links-viewport: 862px;
```

Comment:
Can we use standard Quasar viewports? Or are there unresolvable stuff.
The reason I insist on this, aside reducing complexity, is that it's usually better to have the behaviour/structure changes all to apply on the same breakpoint, otherwise it becomes difficult to reason about and maintain

Reply:
Okay, I had to target some specific viewports at which there is misbehaviour. I'll have to try with quasar viewports which are close to those specifically targeted.